static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -733,7 +733,7 @@ $navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter/2;
     list-style: none;
     white-space: nowrap;
     display: flex;
-        padding-top: $gs-baseline/2;
+    padding-top: $gs-baseline/2;
```

Comment:
good to put spaces around operators (consider `$gs-baseline-2` vs `$gs-baseline - 2`)